test/test_intensify/test_abstract_intensifier.py

```diff
@@ -48,6 +49,33 @@ def setUp(self):
 
         self.logger = logging.getLogger(self.__module__ + "." + self.__class__.__name__)
 
+    def test_get_next_challenger(self):
+        """
+            test get_next_challenger - pick from list/chooser
+        """
+        intensifier = AbstractRacer(
+            tae_runner=None, stats=self.stats, traj_logger=None,
+            rng=np.random.RandomState(12345), deterministic=True, run_obj_time=False,
+            cutoff=1, instances=[1], initial_budget=1, max_budget=3, eta=2)
+
+        # None when nothing to choose from
+        config = intensifier.get_next_challenger(challengers=None, chooser=None, run_history=self.rh)
+        self.assertEqual(config, None)
+
+        # next challenger from a list
+        config = intensifier.get_next_challenger(challengers=[self.config1], chooser=None, run_history=self.rh)
```

Comment:
This test would be more convincing if used more than one configuration, and would run for more than one iteration.